lib/logstash/inputs/kafka.rb

```diff
@@ -212,6 +212,10 @@ def thread_runner(logstash_queue, consumer)
               logstash_queue << event
             end
           end
+          # Manual offset commit
+          if @enable_auto_commit == "false"
+            consumer.commitSync
```

Comment:
Well it's a tradeoff between performance and consistance.
Yet I admit that manually committing doesn't prevent data loss because the commit is done after having placed the event in the the internal logstash queue, not after it's written in Elasticsearch.
Setting the enable_auto_commit to false (disabling auto commit) and not manually committing offsets afterwards is a non sense to me. Because it means each time logstash is restarted, all messages are read again.
I could replace the commit by a commitAsync for better performance